platform/src/abstractions/EducationBenchmarking.Platform.Infrastructure/EducationBenchmarking.Platform.Infrastructure.csproj

```diff
@@ -9,7 +9,9 @@
 
     <ItemGroup>
       <PackageReference Include="Azure.Search.Documents" Version="11.5.1" />
-      <PackageReference Include="Microsoft.Extensions.Options" Version="8.0.0" />
+      <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.37.0" />
+      <PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
+      <PackageReference Include="System.Linq.Async" Version="6.0.1" />
     </ItemGroup>
 
 </Project>
```

Comment:
Should we be looking for the latest version for a particular release rather than keeping them fixed? e.g. 

`<PackageReference Include="Azure.Search.Documents" Version="11.*" />` 
vs
`<PackageReference Include="Azure.Search.Documents" Version="11.5.1" />1

Reply:
Constantly using latest can cause instability in the code base. 

We should look to pin versions where possible, and manager version updates as and when needed.